cogs/discord_datamining.py

```diff
@@ -74,18 +74,30 @@ async def commit_poster_loop(self, embed_title:str, api_url:str, repo_url:str):
                 data = await r.json()
             comment_text.extend([(i['commit_id'], re.sub(r"^## (.+)", r"**\1**", i['body'])) for i in data])
         self.logger.info(f"Logging info of {len(comment_text)} comments")
-
-        # Format it all into an embed
+        #get the description
+        pog =  '\n'.join([f"{i['commit']['message']} - [Link]({repo_url}commit/{i['sha']})" for i in new_commits])
+        if len(pog)>2048:
+            self.logger.info(f"So it turns out our description is longer than 2048 and this isnt really poggers, we need to make it in multiple embeds. It was {len(pog)}")
```

Comment:
This kinda overbloats it, than again it really doesn’t matter as kae is probably the only ever who will see this lol

Reply:
i made it more for like debugging
kae could easily remove it

Reply:
Log line is fine - that's the point of log lines.